local/compose/scan_suggest.go

```diff
@@ -0,0 +1,90 @@
+/*
+   Copyright 2020 Docker Compose CLI authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package compose
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	pluginmanager "github.com/docker/cli/cli-plugins/manager"
+	"github.com/docker/cli/cli/command"
+	"github.com/docker/compose-cli/api/config"
+)
+
+func displayScanSuggestMsg(ctx context.Context, builtImages []string) {
+	if len(builtImages) <= 0 {
+		return
+	}
+	if os.Getenv("DOCKER_SCAN_SUGGEST") == "false" {
+		return
+	}
+	if !scanAvailable() || scanAlreadyInvoked(ctx) {
+		return
+	}
+	commands := []string{}
+	for _, image := range builtImages {
+		commands = append(commands, fmt.Sprintf("docker scan %s", image))
+	}
+	allCommands := strings.Join(commands, ", ")
```

Comment:
Maybe output one scan command per line. Having all commands on one line would be confusing (unless we can pass all images to one scan command).

Reply:
updated with this